roles/lnls-ans-role-epics/meta/main.yml

```diff
@@ -4,7 +4,7 @@ galaxy_info:
   description: Ansible role to install EPICS base and extra packages
   company: CNPEM - LNLS
   license: BSD
-  min_ansible_version: 2.4
+  min_ansible_version: 2.6
```

Comment:
i am thinking about including ansible in the deploy, so that operators can use it to start/stop IOCs from any desktop computare. i think it would be useful if we could use a version that is compatible with de default versions in debian9 and debian10. is this the case? is this possible?

Reply:
Yes. When installing ansible with `pip` we can use the latest one, which is 2.9.

Reply:
I think we can always install the latest ansible version through pip. Or do you think it's better to install a specific one?

Reply:
i noticed native apt ansible version in some distros were older than the version we have been using. i did not think about pip. in this case we can use the newest, no problem!

Reply:
Nice! And yes. I've seen ansible version older than 2.6 in some distros.